dev/Interop/StoragePickers/PickerLocalization.cpp

```diff
@@ -9,14 +9,119 @@
 #include "PickerLocalization.h"
 #include <winrt\base.h>
 #include <winrt\Microsoft.Windows.ApplicationModel.Resources.h>
-#include <iostream>
 
+// find framework package related code from https://github.com/microsoft/microsoft-ui-xaml/blob/main/src/controls/dev/ResourceHelper/ResourceAccessor.cpp
+// and https://github.com/microsoft/microsoft-ui-xaml/blob/main/src/controls/dev/dll/SharedHelpers.cpp
+
+namespace {
+    constexpr wchar_t c_reunionPackageNamePrefix[] = L"Microsoft.WindowsAppRuntime";
+    constexpr int c_reunionPackageNamePrefixLength = ARRAYSIZE(c_reunionPackageNamePrefix) - 1;
+
```

Comment:
ProjectReunion is the old name - let's use WindowsAppSDK for build time considerations and WindowsAppRuntime for runtime.
```suggestion
    constexpr wchar_t c_WindowsAppRuntimePackageNamePrefix[] = L"Microsoft.WindowsAppRuntime";
    constexpr int c_WindowsAppRuntimePackageNamePrefixLength = ARRAYSIZE(c_WindowsAppRuntimePackageNamePrefix) - 1;

```